src/pages/RepoPage/SettingsTab/tabs/GeneralTab/GeneralTab.jsx

```diff
@@ -1,19 +1,30 @@
+import { useParams } from 'react-router-dom'
+
 import { useRepoSettings } from 'services/repo'
+import { TierNames, useTier } from 'services/tier'
+import { useFlags } from 'shared/featureFlags'
 
 import DangerZone from './DangerZone'
 import DefaultBranch from './DefaultBranch'
-import Tokens from './Tokens'
+import { Tokens, TokensTeam } from './Tokens'
 
 function GeneralTab() {
   const { data } = useRepoSettings()
+  const { provider, owner } = useParams()
+  const { data: tierData } = useTier({ provider, owner })
+
+  const { multipleTiers: isTeamTier } = useFlags({
+    multipleTiers: false,
+  })
```

Comment:
Left over from a previous PR, but I think this naming alias may be a bit dangerous as it's not descriptive of the flag and can easily be confused with creating a variable based off of a conditional check.

Reply:
Yeaah sounds good, I'll change it